src/services/users.services.js

```diff
@@ -0,0 +1,50 @@
+// const { ApiError } = require('../exceptions/ApiError');
+const { User } = require('../models/user');
+const { hashPassword } = require('../utils/password/hashPassword');
+
+const getUsers = async () => {
+  const users = await User.findAll();
+
+  return users;
+};
+
+const getUserByEmail = async (email) => {
+  const user = await User.findOne({ where: { email } });
+
+  return user;
+};
+
```

Comment:
Consider renaming `getUsersById` to `getUserById` to accurately reflect that it retrieves a single user by ID, not multiple users.